bin/probot-receive.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+// Usage: probot receive -e push -p path/to/payload app.js
+
+require('dotenv').config()
+
+const path = require('path')
+const uuid = require('uuid')
+const program = require('commander')
+
+const { findPrivateKey } = require('../lib/private-key')
+const { createProbot } = require('../')
+
+program
+  .usage('[options] [path/to/app.js...]')
+  .option('-e, --event <event-name>', 'Event name', process.env.GITHUB_EVENT_TYPE)
+  .option('-p, --payload-path <payload-path>', 'Path to the event payload', process.env.GITHUB_EVENT_PATH)
+  .option('-t, --token <access-token>', 'Access token', process.env.GITHUB_TOKEN)
+  .option('-a, --app <id>', 'ID of the GitHub App', process.env.APP_ID)
+  .option('-P, --private-key <file>', 'Path to certificate of the GitHub App', findPrivateKey)
+  .parse(process.argv)
+
+const githubToken = program.token
+
+if (!program.event || !program.payloadPath || !githubToken) {
+  program.help()
+}
+
+if (githubToken) {
+  process.env.DISABLE_STATS = 'true'
+}
+
+const payload = require(path.join(process.cwd(), program.payloadPath))
+
+const probot = createProbot({
```

Comment:
Should we consider adding a check to see if the app has been created and if not redirect to manifest setup? Like how we do here:
https://github.com/probot/probot/blob/0036fee389148fb7b0b86865f672ffb0de43f0b8/bin/probot-run.js#L33-L44